UnitsNet/UnitsNet.csproj

```diff
@@ -56,5 +56,8 @@
     <EmbeddedResource Remove="UnitDefinitions\**" />
     <None Remove="UnitDefinitions\**" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="..\Common\UnitDefinitions\StellarLuminosity.json" />
+  </ItemGroup>
```

Comment:
Remove this ItemGroup. In fact you can just revert all changes to this file.

Reply:
Removed